ReSharper.FSharp/src/FSharp.TypeProviders.Protocol/src/Cache/IProvidedEntitiesCache.cs

```diff
@@ -73,8 +73,8 @@ public T[] GetOrCreateBatch(TKey[] keys, int typeProviderId, TParam parameters)
           foreach (var (key, i) in group)
           {
             if (!KeyHasValue(key)) continue;
-            Assertion.Assert(Entities.TryGetValue(key, out var providedEntity),
-              "Possible concurrent Remove(typeProviderId) call");
+            var hasValue = Entities.TryGetValue(key, out var providedEntity);
+            Assertion.Assert(hasValue, "Possible concurrent Remove(typeProviderId) call");
```

Comment:
Not really an issue, but wouldn't `AssertNotNull(providedEntity, ...)` be a simpler check here?